utils/mdx.ts

```diff
@@ -55,7 +55,16 @@ const remarkPlugins: PluggableList = [
 ];
 
 const rehypePlugins: PluggableList = [
-  rehypeHighlight,
+  [
```

Comment:
This demo application is just for demonstrating `next-mdx-remote-client` features. I do not plan to change code highlighting plugin. There is one more reason that I am also the author of `rehype-highlight-code-lines` and wanna see it is working with `rehype-highlight`.